content/docs/user-guide/managing-external-data.md

```diff
@@ -61,6 +61,12 @@ types:
    moves a local file to the external location. This produces an external output
    in `dvc.yaml`.
 
+> 🔒 Note that for locations that require authentication there may be extra
+> steps required. One way is to configure the remote accordingly after step 1,
```

Comment:
Not clear still. " ... there may be extra steps required. One way ..." feels like some explanation of the "extra step" is missing. 

Also, almost all locations require auth. I would say this about some specific auth types.

Reply:
> feels like some explanation of the "extra step" is missing.

That's what the last sentence, "See `dvc remote modify` for more alternatives," is for.

> almost all locations require auth. I would say this about some specific auth types.

Agree. But this PR is meant as a patch: previously it was only mentioned in SSH and now I'm generalizing the note to apply to every type for now. Since this whole `--external` thing is strongly discouraged and the guide basically needs rewriting too, is it worth adding more complication to it now?

Reply:
> That's what the last sentence, "See dvc remote modify for more alternatives," is for.

should it go before the "one way" then? Otherwise it looks like it is broken

> 'm generalizing the note to apply to every type for now

I understand. It becomes not a patch though but rather an improvement. The phrase `locations that require authentication` still looks too general. It happens only to certain types of auth. Easy fix is "Depending on the remote storage and authentication type, there may be ... You can use special remote:// ... to provide authentication and other remote settings via DVC config. See ..."

Reply:
> should it go before the "one way" then?

Moved.

> "locations that require authentication" still looks too general. It happens only to certain types of auth.

That part is general, but "there **may** be extra steps" narrows it down ("may").
But I rephrased the example along the lines of your suggestion (clarified step 1 too).